packages/project/src/index.ts

```diff
@@ -26,6 +26,7 @@ export function *loadConfigFile(configFilePath: string): Operation<ProjectOption
 
 export type ProjectOptions = {
   port: number;
+  showTree: boolean;
```

Comment:
I wonder if `showTree` should be a project option. On the one hand, I like the idea that there is a 1:1 correspondence between CLI options and project config options, but it does seem odd that something that is only used by the CLI is being passed by the orchestrator. On the other hand, it also allows making the option persistent.

Reply:
I didn't know how else to get the option passed down to where we output to the console. Open to suggestions for sure!

Reply:
Yeah, I'm thinking purely from a philosophical sense. Like, it would be awesome if every single config option could be overridden from the command line, so this probably is the best approach? Let's roll with this.